conjure-java-core/src/test/resources/dangerous-name-service.yml

```diff
@@ -52,3 +52,12 @@ services:
             markers:
               - Safe
         returns: string
+      noContext:
+        http: POST /no/context
+        args:
+          requestContext: string
+      context:
+        http: POST /context
+        args:
+          requestContext: string
```

Comment:
This is just to test that we can generate the code properly with a request context, right?

Reply:
That's correct, we're testing that an existing parameter with the expected `ServerRequestContext` parameter name compiles properly.

The endpoint above tests that when you have an existing variable named `requestContext` which would clash with the special parameter, it's _not_ renamed unnecessarily when there's no `ServerRequestContext` parameter.